packages/playwright-test/types/test.d.ts

```diff
@@ -3067,19 +3094,9 @@ export type Expect = {
   extend(arg0: any): void;
   getState(): expectType.MatcherState;
   setState(state: Partial<expectType.MatcherState>): void;
-  any(expectedObject: any): AsymmetricMatcher;
-  anything(): AsymmetricMatcher;
-  arrayContaining(sample: Array<unknown>): AsymmetricMatcher;
-  objectContaining(sample: Record<string, unknown>): AsymmetricMatcher;
-  stringContaining(expected: string): AsymmetricMatcher;
-  stringMatching(expected: string | RegExp): AsymmetricMatcher;
-  /**
-   * Removed following methods because they rely on a test-runner integration from Jest which we don't support:
-   * - assertions()
-   * - extractExpectedAssertionsErrors()
-   * – hasAssertions()
-   */
-};
+} & BaseExpect &
```

Comment:
BaseExpect repeats the 3 lines above, let's pick one. I prefer inlining instead of a separate type :)